packages/cxl-ui/src/media-context.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from '@lit/context';
+
+export const mediaContext = createContext(Symbol('media-context'));
```

Comment:
Should it be a more specific `mediaQueryContext` maybe?

"Media" alone seems like could be related to video, or whatnot more broad concept.